WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderWebView.java

```diff
@@ -97,10 +97,9 @@ private void init(Context context) {
             this.setWebViewClient(new ReaderWebViewClient(this));
             this.getSettings().setUserAgentString(WordPress.getUserAgent());
 
-            // Adjust content font size on APIs 19 and below as those do not do it automatically.
             // If fontScale is close to 1, just let it be 1.
             final float fontScale = getResources().getConfiguration().fontScale;
-            if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.KITKAT && ((int) (fontScale * 10000)) != 10000) {
```

Comment:
Should we not remove the whole if branch here? Since it adjusts the content font size on APIs 19  and lower when the font scale is not 1.